client/js/screens/Event/Event.js

```diff
@@ -0,0 +1,187 @@
+                  />
+                </View>
+              </View>
+            </View>
+          </View>
+
+          <View style={styles.eventScreenInfo}>
+            <Text style={styles.infoText}>Date / Time</Text>
+            <View
+              style={{
+                ...styles.eventTextInput,
+                height: 36,
+                flexDirection: 'row',
+                alignItems: 'center',
+              }}>
+              <Image
+                style={styles.eventTextFields}
+                source={require('../../assets/images/icFilterDate.png')}
+              />
+              <Text style={{fontSize: 12}}>{eventInfo.date}</Text>
+            </View>
+
+            <Text style={styles.infoText}>Location</Text>
+            <View
+              style={{
+                ...styles.eventTextInput,
+                height: 36,
+                flexDirection: 'row',
+                alignItems: 'center',
+              }}>
+              <Image
+                style={styles.eventTextFields}
+                source={require('../../assets/images/icFilterLocation.png')}
+              />
+              <Text style={{fontSize: 12}}>{eventInfo.address}</Text>
+            </View>
+
+            <View style={styles.eventDetails}>
+              <View style={{...styles.cols2, ...styles.firstColumn}}>
+                <Text style={styles.infoText}>Difficulty</Text>
+                <View
+                  style={{
+                    ...styles.eventTextInput,
+                    height: 36,
+                    flexDirection: 'row',
+                    alignItems: 'center',
+                  }}>
+                  <Image
+                    style={styles.eventTextFields}
+                    source={require('../../assets/images/icFilterDifficulty.png')}
+                  />
+                  <Text style={{fontSize: 12}}>{eventInfo.difficulty}</Text>
+                </View>
+              </View>
+
+              <View style={{...styles.cols2, ...styles.firstColumn}}>
+                <Text style={styles.infoText}>Language</Text>
+                <View
+                  style={{
+                    ...styles.eventTextInput,
+                    height: 36,
+                    flexDirection: 'row',
+                    alignItems: 'center',
+                  }}>
+                  <Image
+                    style={styles.eventTextFields}
+                    source={require('../../assets/images/icFilterLanguage.png')}
+                  />
+                  <Text style={{fontSize: 12}}>{eventInfo.language}</Text>
+                </View>
+              </View>
+            </View>
+
+            <View>
+              <Text style={styles.eventInfoTitle}>Event Details</Text>
+
+              <Text style={styles.eventInfoDescription}>
+                {eventInfo.details.replace(/\\n/g, '\n')}
+              </Text>
+              {/* <Text style={styles.eventReadMore}>(Read More)</Text> */}
```

Comment:
Please remove commented code if not needed.

Reply:
Need this for later.